integrated_channels/integrated_channel/models.py

```diff
@@ -328,6 +379,7 @@ class ContentMetadataItemTransmission(TimeStampedModel):
 
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, on_delete=models.CASCADE)
     integrated_channel_code = models.CharField(max_length=30)
+    plugin_configuration_id = models.PositiveIntegerField(blank=True, null=True)
```

Comment:
I think we ran some issues before with biginteger firld and index name length?

Reply:
ok this one it looks like https://edx-internal.slack.com/archives/C014MDBL8JW/p1643827874773499?thread_ts=1643826987.458379&cid=C014MDBL8JW

Reply:
I found this field type used elsewhere for a non-foreign-key id join column. i can take a peek to see if we've got any bugs/docs on this...